llvm/include/llvm/ADT/EquivalenceClasses.h

```diff
@@ -178,6 +178,9 @@ class EquivalenceClasses {
   member_iterator member_end() const {
     return member_iterator(nullptr);
   }
+  auto members(iterator I) const {
```

Comment:
Could you spell out the type here as is generally preferred in LLVM unless not possible otherwise?

Reply:
In this case, I do not think it's worth spelling out the complex type of `make_range`. STLExtras also follows the auto-return practice when make_range is involved, in some cases:

```
template <typename T> auto drop_begin(T &&RangeOrContainer, size_t N = 1) {
  return make_range(std::next(adl_begin(RangeOrContainer), N),
                    adl_end(RangeOrContainer));
}
```